static/css/mozsearch.css

```diff
@@ -497,15 +506,15 @@ tr.after-context-line + tr.before-context-line {
 .bubble a:visited,
 .result-head a:link,
 .result-head a:visited {
-  color: #0095dd;
+  color: var(--link-color);
 }
 .breadcrumbs a:hover,
 .result-head a:hover {
-  color: #00539f;
+  color: var(--link-hover-color);
```

Comment:
https://github.com/mozsearch/mozsearch/pull/468 has some additional clean-ups here, might be worth incorporating, but happy to do them in a follow-up instead.

Reply:
If you're working on this actively please feel free to even reject this and e.g. only land the expando change. I might not check back here for a while once I jump into meetings.

Reply:
Ok, I'll land that and rebase your branch to land the expando change. That way we probably get it on the next indexer run.